ansible/playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
I'd like to see this go to openshift-ansible-contrib, we've had some community contributors that have added features to this.